


- Music: 
Faszo by Mini Vandals

- Movies: 
explications_segal_1.mov
explications_segal_2.mov
explications_segal_3.mov



--- Vidéo #1 Stage d'observation du 20 au 22 décembre 2023 au sein de France Médias Monde (Français)
--- https://youtu.be/h1E_QOrYdHs



--- Vidéo #2 Stage d'observation du 20 au 22 décembre 2023 au sein de France Médias Monde (Français)
--- https://youtu.be/jFUEhhPX2NI


Tout le code se trouve sur le compte GitHub à cette adresse
https://github.com/bflaven/learning_python_for_all/tree/main/000_jupyter_notebooks

- Utilisation d'un Jupyter notebook 
- Installation de Ollama sur le poste de travail avec initiation aux prompts sur llama2 au 21/12/23

--- Vidéo #3 Stage d'observation du 20 au 22 décembre 2023 au sein de France Médias Monde (Français)
--- https://youtu.be/rhvjFIb6YhQ


Jour 3
Introduction plus avancée aux concepts de programmation avec la gestion des environnements via Anaconda et des packages et utilisation de requêtes avancées sur llama2 via Ollama

Tout le code se trouve sur le compte GitHub à cette adresse
https://github.com/bflaven/learning_python_for_all/tree/main/000_jupyter_notebooks

- Utilisation des packages Python et des environnements de développement avec Anaconda
- Utilisation du LLM llama2 via de Ollama sur le poste de travail
- Découverte de la Création d'image en programmation Python

Qu'est-ce qu'un grand modèle de langage (LLM) ?
LLM : Large Language Model
Un LLM est un modèle de langage étendu (LLM) est un type de programme d'intelligence artificielle (IA ) capable de reconnaître et de générer du texte, entre autres tâches. Les LLM sont formés sur d'énormes ensembles de données - d'où le nom "large." Les LLM reposent sur l'apprentissage automatique : en particulier, un type de réseau neuronal appelé modèle de transformateur.

- Visuel: 
https://agglo2b.fr/images/social/teaser_stage3eme.jpg

--- tags
stage, 3ème, première expérience, opportunité, stage d'observation, réussir, bons réflexes à adopter, ,  prendre des notes, poser des questions, être attentif, attentif ensemble, python, LLM, IA, sublime, visual code, editeur, programmation, initiation, prompts, ollama, data science, llama2, explication, tutoriel, prise en main

- description
                Cette vidéo fait partie d'une série "Stage d'observation du 20 au 22 décembre 2023 au sein de France Médias Monde (Français)"


                Check my blog @ http://www.flaven.fr
                Check my Github @ http://github.com/bflaven/

                Music
                Faszo by Mini Vandals
                Thanks to Mini Vandals
                Music by Mini Vandals
                
                Plus que jamais, prenez soin de vous et des vôtres,
                Take care of yourself and the ones you loved.

                © Flaven.fr Dec 2023
